src/api/Alert/types.ts

```diff
@@ -244,3 +244,46 @@ export type AlertPreview = Array<{
   edges: LkEdge[];
   columns: Array<string | number>;
 }>;
+
+export interface ICaseColumn {
+  type: AlertColumnType;
+  columnValue: string | number;
+  columnTitle: string;
+}
+
+export enum FullCaseListSortProperties {
```

Comment:
@maxime-allex @mzorkany do you think we could align the FullCaseListSortProperties with the IFullCase property keys?
so for exemple we should not have assignedUser in one and assignee in the other. Also, createdAt and creationDate.
For me, it should be the same names.
(And incidentally, it makes it easier for me to do the sorting. If the user clicks on 1 column, I get the column key and send it to the backend. I do not need to map it to a FullCaseListSortProperties)
wdyt?
I would even go further and propose to make, when possible, the keys of the IFullCase interface one of the FullCaseListSortProperties value. In IFullCase:
`[FullCaseListSortProperties.CASE_ID]: number`
It makes it easier to maintain if those value change in the futur!